scripts/myMangaDex.js

```diff
@@ -45,7 +46,8 @@ class MyMangaDex {
 		if ((this.pageUrl.indexOf(urls.follows) > -1 && this.pageUrl.indexOf("/manga/") == -1) ||
 			(this.pageUrl.indexOf(urls.group) > -1 && (this.pageUrl.indexOf("/chapters/") > -1 || (this.pageUrl.indexOf("/manga/") == -1 && this.pageUrl.indexOf("/comments") == -1))) ||
 			(this.pageUrl.indexOf(urls.user) > -1 && (this.pageUrl.indexOf("/chapters/") > -1 || this.pageUrl.indexOf("/manga/") == -1))) {
-			this.chapterListPage();
+			this.pageType = "chapterList";
+			this.chapterListPage(this.pageUrl.indexOf(urls.follows) > -1);
```

Comment:
Currently I only fetch on the follows list, do you want to do it on other pages as well?

Reply:
No this should be enough, other pages are rarely used (I believe), and also for less time, if somebody request it for other pages we could just enabled it but it's not useful in my opinion.